src/path.cc

```diff
@@ -28,13 +18,13 @@ std::string NormalizeString(const std::string_view path,
   for (size_t i = 0; i <= path.size(); ++i) {
     if (i < path.size()) {
       code = path[i];
-    } else if (IsPathSeparator(code)) {
+    } else if (code == std::filesystem::path::preferred_separator) {
```

Comment:
These changes are wrong on Windows, where there are two valid path separators.

Reply:
It would make sense to keep `IsPathSeparator(...)` as a separate function and update the existing checks there to use `std::filesystem::path::preferred_separator`. Doing so will allow minimizing the changes in this file while accounting for the multiple valid path separators on Windows.